hawk/app/assets/javascripts/module/resources.js

```diff
@@ -711,7 +711,7 @@ $(function() {
         events: {
           'click .delete': function (e, value, row, index) {
             e.preventDefault();
-            $.hawkDeleteOperation(row.id, [$(this).attr('href'), ".json"].join(""));
+            $.hawkDeleteOperation(row.id, $(this).attr('href'));
```

Comment:
On the "Edit Configuration" page the delete operation through the shortcut didn't worked anymore after allowing the dot's in the url. Turned out that the json extension was inheritedto the ID now which leaded to an 404.

Reply:
Oh, this might be a problem in several places, we rely on the ".json" extension to work for the dashboard as well for example...

Reply:
But for the DELETE requests, I don't see any reason to include the ".json" extension... it only makes sense for GET requests...